sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/simulator/_model_tools/_identity_manager.py

```diff
@@ -14,7 +14,7 @@
 from azure.core.credentials import AccessToken, TokenCredential
 from azure.identity import DefaultAzureCredential, ManagedIdentityCredential
 
-AZURE_TOKEN_REFRESH_INTERVAL = 600  # seconds
+AZURE_TOKEN_REFRESH_INTERVAL = os.getenv("AZURE_TOKEN_REFRESH_INTERVAL", 600) # token refresh interval in seconds
```

Comment:
could we refresh token based on the token exipration time?

https://github.com/microsoft/promptflow/blob/f08e57614f8655582abd7d25097aacd5b05ab22b/src/promptflow-azure/promptflow/azure/_utils/_token_cache.py#L21

Reply:
I think it happens [here](https://github.com/Azure/azure-sdk-for-python/blob/bc9e0d322cdcc6483ed8676ad5bd9ee2862bd345/sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/simulator/_model_tools/_identity_manager.py#L131 )
It just uses the `AZURE_TOKEN_REFRESH_INTERVAL`